clib/src/main/resources/scala-native/epoll.c

```diff
@@ -0,0 +1,26 @@
+#ifdef __linux__
```

Comment:
I'll address my perennial concerns about the location
and nature of libc in the close related, in content & time,
PR #4625.  That will consolidate the discussion.

TL; DR 

I believe the location must be improved and will
probably submit a follow-on PR to do so.  

That SN housekeeping detail should not, IMO, delay
getting these definitions into the hands of someone 
using them to solve much bigger concerns.